Dockerfile

```diff
@@ -267,8 +267,6 @@ RUN --mount=type=tmpfs,target=/android \
     fi \
     && yes | $ANDROID_HOME/cmdline-tools/bin/sdkmanager $SDK_MANAGER_PROXY_OPTIONS --sdk_root=$ANDROID_HOME "platform-tools" "cmdline-tools;latest"
 
-COPY docker/android.sh /etc/profile.d
```

Comment:
This becaume unneeded when `android` image / layer was taken into use? Would it make sense to squash this as well?

Reply:
i do prefer keep in a separate commit. Was squashed before, and i was told to create separate commits

Reply:
Ok, I didn't review previous discussions and don't want to introduce back and force.